recipes/pykdtree/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [[ $(uname) == Darwin ]]; then
+    USE_OMP=0
```

Comment:
I don't really remember why I had to do this. @jakirkham you are the OS X expert can you take a look and see if this is still necessary and/or if there is a way around it?